src/xdocs/config_regexp.xml

```diff
@@ -145,43 +145,43 @@
         <source>
           // This code is copyrighted
         </source>
-        <p> The check. </p>
+          <p> The check. </p>
         <source>
           &lt;module name="Regexp"&gt;
-          &lt;property name="format" value="// This code is copyrighted"/&gt;
+            &lt;property name="format" value="// This code is copyrighted"/&gt;
```

Comment:
goo, and now remove all leading spaces in SOURCE tag.

Reply:
@romani Does the source block needs to be shifted to the extreme right?

Current Code (From line 1054) :
```
        <p>
          To configure the check to find case-insensitive occurrences of
          &quot;debug&quot;:
        </p>
        <source>
          &lt;module name=&quot;RegexpSinglelineJava&quot;&gt;
              &lt;property name=&quot;format&quot; value=&quot;debug&quot;/&gt;
              &lt;property name=&quot;ignoreCase&quot; value=&quot;true&quot;/&gt;
          &lt;/module&gt;
        </source>
```
After change (According to my understanding)
```
        <p>
          To configure the check to find case-insensitive occurrences of
          &quot;debug&quot;:
        </p>
<source>
  &lt;module name=&quot;RegexpSinglelineJava&quot;&gt;
      &lt;property name=&quot;format&quot; value=&quot;debug&quot;/&gt;
      &lt;property name=&quot;ignoreCase&quot; value=&quot;true&quot;/&gt;
  &lt;/module&gt;
</source>
```
Please let me know if I am on the right track for this change? Thanks

Reply:
@varunu28 I believe he just wants the excess indentation inside the source tag removed.
So this:
````
        <source>
&lt;module name=&quot;RegexpSinglelineJava&quot;&gt;
  &lt;property name=&quot;format&quot; value=&quot;debug&quot;/&gt;
  &lt;property name=&quot;ignoreCase&quot; value=&quot;true&quot;/&gt;
&lt;/module&gt;
        </source>
````
This is because users see whats in the source tag, including the excess indentation, but they don't see the outer tags.

Reply:
SOURCE tag should stay as is, content of SOURCE should be moved to left.